src/createSchemaForm.tsx

```diff
@@ -443,14 +494,50 @@ export function createTsForm<
         form.reset(defaultValues);
       }
     }, []);
-    const { control, handleSubmit, setError, getValues } = _form;
+    const { handleSubmit, setError } = _form;
     const submitter = useSubmitter({
       resolver,
       onSubmit,
       setError,
     });
     const submitFn = handleSubmit(submitter.submit);
 
+    return (
+      <FormProvider {..._form}>
+        <SubmitterContextProvider {...submitter}>
+          <FormComponent {...formProps} onSubmit={submitFn}>
+            <FormFragment
+              {...({
+                schema,
+                props,
+                renderAfter:
+                  renderAfter && (() => renderAfter({ submit: submitFn })),
+                renderBefore:
+                  renderBefore && (() => renderBefore({ submit: submitFn })),
+                children: CustomChildrenComponent,
+              } as any)}
```

Comment:
I couldn't for the life of me get it to accept these props even though I'm certain they're right..